lang/extract_json_strings.py

```diff
@@ -1169,25 +1217,27 @@ def extract(item, infilename):
                          .format(name, special_attack.get("spell_id")), **kwargs)
                 wrote = True
     if "footsteps" in item:
-       writestr(outfile, item["footsteps"], **kwargs)
-       wrote = True
-    if not wrote and not "copy-from" in item:
+        writestr(outfile, item["footsteps"], **kwargs)
+        wrote = True
+    if not wrote and "copy-from" not in item:
         if not warning_supressed(infilename):
             print("WARNING: {}: nothing translatable found in item: {}".format(infilename, item))
 
+
 def is_official_mod(full_path):
-    for i in official_mods:
+    for i in official_mods:  # FIXME: official_mods undefined
```

Comment:
@BrettDong Can you please help with that? Maybe you remember the context and can fix it quickly?

Reply:
Ok, looks like it was never used, I'm removing it.

Reply:
I have no idea of this function either.